### PR TITLE
Black dashboard items and Full Screen

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Adds support for black dashboard panels
- Adds full screen button

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)

### CODE REVIEW

**Comment by mariusandra** [comment]
This PR could already be merged to already some more dashboard issues fixed in master.

File: frontend/src/scenes/dashboard/dashboardLogic.js [hunk 1]
```diff
@@ -138,12 +150,68 @@ export const dashboardLogic = kea({
...
+                const newItem = dashboardId ? { ...rest, dashboard: dashboardId } : { ...rest }
+                const addedItem = await api.create('api/dashboard_item', newItem)
+
+                if (move) {
+                    const deletedItem = await api.update(`api/dashboard_item/${item.id}`, { deleted: true })
+                    dashboardsModel.actions.updateDashboardItem(deletedItem)
+
+                    const toastId = toast(
+                        <div>
+                            Panel moved to dashboard.&nbsp;
```
Comment (timgl) [inline frontend/src/scenes/dashboard/dashboardLogic.js:181]: Would be nice to have a link to that dashboard here

**Review by timgl — APPROVED** [review]

File: frontend/src/scenes/dashboard/dashboardLogic.js (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/dashboard/dashboardLogic.js]: added!

**Comment by timgl** [comment]
LGTM

**Comment by mariusandra** [comment]
Thanks! I had to add one more rather complicated piece of code (https://github.com/PostHog/posthog/pull/753/commits/57a3f7b7e9d21a0ba6b0ae871840e72d2c517041) to fix alignment issues. 

This is one of the more important points from #752.

Rather than explain, here are screencasts:

Before:
screencast 2020-05-14 00-15-03

After:
screencast 2020-05-14 00-10-26

The "before" behaviour was there no matter if you added the chart from the funnel/trends page, copied/moved the element or just duplicated it. I think it was totally worth the effort :).

If looks good, I'll merge.

Just touch input and edit item to go and the dashboards are done.

**Comment by timgl** [comment]
LGTM

**Pull request merged**